client/executor/src/integration_tests/mod.rs

```diff
@@ -40,13 +101,14 @@ fn call_in_wasm<E: Externalities>(
 	code: &[u8],
 	heap_pages: u64,
 ) -> crate::error::Result<Vec<u8>> {
-	crate::call_in_wasm::<E, sp_io::SubstrateHostFunctions>(
+	crate::call_in_wasm::<E, HostFunctions>(
```

Comment:
Use always the host functions with the fake functions.

And revert the `true` below to `false`.